tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -3207,6 +3251,60 @@ def test_transaction_user_misery(self) -> None:
             },
         }
 
+    def _setup_user_misery(self, spec: OnDemandMetricSpec):
+        events = [
+            ("one", 300),
+            ("two", 300),
+            ("one", 3000),
+            ("two", 3000),
+            ("three", 400),
+            ("four", 4000),
+        ]
+        for index, event in enumerate(events):
+            email = f"{event[0]}@example.com"
+            tags = {"user.email": email}
+            if event[1] > 300:
+                tags["satisfaction"] = "frustrated"
```

Comment:
@k-fish I don't think I'm doing this quite right since if I comment it out it will make no difference.

Anything in this function maybe the reason why `miserable_users` is returning 6 consistently.